### PR TITLE
Add alt and title

### PR DESCRIPTION
**Pull request by tanvi0012**
Add alt text for user avatar and title for collapsible nav links

### CODE REVIEW

File: templates/subscriptions_page.hbs [hunk 1]
```diff
@@ -15,7 +15,7 @@
         {{/if}}
         <li>{{link 'contributions'}}</li>
         <li class="current">
-          <a href="{{page_path 'following'}}" aria-current="page">{{ t 'following' }}</a>
+          <a href="{{page_path 'following'}}" aria-current="page" title="{{ t 'following' }}">{{ t 'following' }}</a>
```
Comment (anpa) [inline templates/subscriptions_page.hbs]: What's the motivation for this change @tanvi0012?

File: templates/subscriptions_page.hbs [hunk 2]
```diff
@@ -61,7 +61,7 @@
             <tr>
               <td>
                 {{#is type 'user'}}
-                  <img class="user-avatar" src="{{avatar_url}}" alt="">
+                  <img class="user-avatar" src="{{avatar_url}}" alt="User Avatar">
```
Comment (anpa) [inline templates/subscriptions_page.hbs]: Same here, what's the motivation for this change? Given this image is adjacent to the name of the user, it should act as decorative and therefore shouldn't need an `alt` attribute.

**Comment by augustocravosilva** [comment]
Closing this PR as stale.
